course2/public/guess.js

```diff
@@ -0,0 +1,206 @@
+// guess the word module
+let guessTheWord = function () {
+	// "config" options
+	const MAX_FAILS = 6;
+	const BONUS_PER_LETTER = 5; // bitcoins
+	const chosenWord = [];
+
+
+	let updateFailTriesCount = function (fails, letters) {
+		$('#staticTries').val(fails);
+		$('#letterAttemps').val(letters);
+	};
+
+
+	let getRandomWords = function () {
+		// randomly pick a word from the word list
+		
+		let randomCategoryKey = Math.floor(Math.random() * Object.keys(wordsToGuess).length), //get a random key between 0 and object length
+			randomWordKey = Math.floor(Math.random() * Object.values(Object.values(wordsToGuess)[randomCategoryKey])[0].length), //get a random key between 0 and the length of words array from the category
+			randomCategory = Object.keys(Object.values(wordsToGuess)[randomCategoryKey]).join(''), //get the category name which was choosed randomly
+			randomWord = Object.values(Object.values(wordsToGuess)[randomCategoryKey])[0][randomWordKey].split(' '); //get the word from that cathegory which was choosed randomly
+		chosenWord.push([...randomWord]);
```

Comment:
If `chosenWord` is not `const`, you could do `chosenWord = randomWord` and replace all the `chosenWord[0]` in the rest of the code with `chosenWord`

Reply:
Done!